### PR TITLE
Fix/866 cant submit seedlot registration with parent trees outside of the orchard on smp mix tab

### LINKED ISSUE #866: Can't submit the submit seedlot registration with parent trees outside of the orchard on SMP Mix tab
**Describe the Bug**
When trying to submit the seedlot registration with a valid parent tree, that is not on the selected orchard but is of the same species, set on the third tab (SMP Mix Calculation), a 404 error is thrown to the user. On the logs a warning message saying "SeedlotParentTree was not found!" is also thrown.

**Expected Behaviour**
Should submit successfully.

**Actual Behaviour**
404 error thrown to the user.

**Steps To Reproduce**
Steps to reproduce the behaviour:
1. Create a seedlot of species FDC.
2. Go to the seedlot registration form.
3. Fill steps 1, 2, 3 and 6 as you wish.
4. Go to step 4 and select orchard `134 - MT. NEWTON - S - PRD`, then fill the rest of the mandatory fields.
5. Go to step 5 and fill the fields on tabs 1 and 2 as you wish
6. On tab 3 (SMP Mix), set parent trees 33 and 34 and set a random valid value for the volume field.
7. Got to step 6 and try to submit
8. See error

**Screenshots**
Image
Image

**Additional context**
This is happening due to the changes when fixing the issue with submitting the correct values for the SMP data.

### PR DESCRIPTION
**Pull request by mgaseta**
# Description
Closes #866

#### New
- Nothing :)

#### Changed
- Removed unnecessary throw
- Renamed pollen count property all over SPAR

#### Removed
- Nothing

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-19-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-19-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-19-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: backend/src/main/java/ca/bc/gov/backendstartapi/service/SeedlotParentTreeSmpMixService.java [hunk 1]
```diff
@@ -72,7 +72,9 @@ private void addSeedlotPtSmpMix(
           seedlotPtFormDto.parentTreeGeneticQualities()) {
         SeedlotParentTree sptEntity = sptMap.get(seedlotPtFormDto.parentTreeId());
         if (Objects.isNull(sptEntity)) {
-          throw new SeedlotParentTreeNotFoundException();
+          // This means that the parent tree is not present in the
+          // selected orchard, so there is no need to include it
+          continue;
```
Comment (RMCampos) [inline backend/src/main/java/ca/bc/gov/backendstartapi/service/SeedlotParentTreeSmpMixService.java:77]: Issue: unused import at line 11 (`SeedlotParentTreeNotFoundException`). Checkstyle is not failing, for some reason :(

**Review by RMCampos — CHANGES_REQUESTED** [review]
Only a small change. Great job! 🥇

File: backend/src/main/java/ca/bc/gov/backendstartapi/service/SeedlotParentTreeSmpMixService.java (hunk 1 above)
Comment (mgaseta) [reply backend/src/main/java/ca/bc/gov/backendstartapi/service/SeedlotParentTreeSmpMixService.java:76]: Fixed here: https://github.com/bcgov/nr-spar/pull/869/commits/0d3427b329202dd978aec3b04792421533ae56e1

**Review by RMCampos — APPROVED** [review]
LGTM! 🎉

**Pull request merged**